mingw-w64-git/PKGBUILD

```diff
@@ -187,7 +187,6 @@ package_git () {
 
   # Git Bash, Git CMD
   install -d -m755 $pkgdir$SHAREDIR
-  install -m755 ../git-for-windows.ico $pkgdir$SHAREDIR
```

Comment:
Uh oh. I missed this when reviewing. This change caused a regression, unfortunately: https://github.com/git-for-windows/git/issues/870